lighthouse-core/audits/dobetterweb/link-blocking-first-paint.js

```diff
@@ -55,31 +56,42 @@ class LinkBlockingFirstPaintAudit extends Audit {
       };
     }
 
-    let totalSpendTime = 0;
-    const results = artifact.items
-      .filter(item => item.tag.tagName === tagFilter)
-      .map(item => {
-        totalSpendTime += item.spendTime;
+    const filtered = artifact.filter(item => item.tag.tagName === tagFilter);
 
-        return {
-          url: item.tag.url,
-          label: `delayed first paint by ${item.spendTime}ms`
-        };
-      });
+    const startTime = filtered.reduce((t, item) => Math.min(t, item.startTime), Number.MAX_VALUE);
```

Comment:
in other words, is it possible for there to be a large difference between the union of time spent with these blocking first paint vs the first `startTime` subtracted from the last `endTime`